Allow using `chrome.storage.local` (#60)
fregante/webext-options-sync: readme.md

Whether info and warnings (on sync, updating form, etc.) should be logged to the console or not.

###### storageType

> Type: `'loacal' | 'sync' | 'managed'`

Reviewer: ```suggestion
Type: `'local' | 'sync' | 'managed'`
```

Default: `sync`

> What storage area type to use (sync storage vs local storage). Sync storage is used by default.

Reviewer 2: ```suggestion
What storage area type to use (sync storage vs local storage). Always prefer `sync` because that's what users most likely expect. Firefox requires [`browser_specific_settings.gecko.id`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings) for the `sync` storage to work locally.
```

Reviewer 2: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings

Reviewer: I think "Always" is slightly too strong; it's also worth mentioning [the storage size constraints](https://developer.chrome.com/docs/extensions/reference/storage/#property-sync).  I wasted a bit of time trying to debug why my extension wouldn't always store data properly before realising the quota was the issue, so it would be great to help others avoid making the same mistake.  Also I guess there may be some privacy concerns around syncing user data without asking.

```suggestion
What storage area type to use (sync storage vs local storage). Sync storage is used by default and is more convenient for users in many situations; however it is subject to much tighter [quota limitations](https://developer.chrome.com/docs/extensions/reference/storage/#property-sync), and may cause privacy concerns if the data being stored is confidential.

Firefox requires [`browser_specific_settings.gecko.id`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings) for the `sync` storage to work locally.
```

Reviewer 2: Always is not strong when coupled with "prefer".

Author: Quota limitations is the reason I'm switching to the local storage, they are considerably more strict for sync
